Source/DafnyDriver/TextLogger.cs

```diff
@@ -42,7 +43,8 @@ public class TextLogger {
           tw.WriteLine(
             $"      {cmd.Tok.filename}({cmd.Tok.line},{cmd.Tok.col}): {cmd.Description}");
         }
-        if (vcResult.CoveredElements.Any()) {
+
+        if (vcResult.CoveredElements.Any() && vcResult.Outcome == ProverInterface.Outcome.Valid) {
```

Comment:
Is this line change a bug fix?

Reply:
I added a test for this. I'm not sure whether I'd consider it a bug fix or just an instance of changing my mind about how the UI should work, but it's good to test either way.